hardware/Honeywell.cpp

```diff
@@ -240,48 +244,91 @@ void CHoneywell::GetThermostatData()
 	}
 
 	int devNr = 0;
-	mDeviceList.clear();
+	mDeviceList.clear(); 
```

Comment:
Please undo whitespace at the end of the line

Reply:
done